data/registry/instrumentation-go-splunkleveldb.yml

```
@@ -15,3 +15,4 @@ authors:
 urls:
   repo: https://github.com/signalfx/splunk-otel-go/tree/main/instrumentation/github.com/syndtr/goleveldb/leveldb/splunkleveldb
 createdAt: 2022-01-20
+isFirstParty: true
```

```suggestion
isFirstParty: false
```

this is not a first party instrumentation